pkgs/tools/networking/openconnect_pa/default.nix

```
@@ -0,0 +1,43 @@
+{ stdenv, fetchFromGitHub, pkgconfig, vpnc, openssl ? null, gnutls ? null, gmp, libxml2, stoken, zlib, autoreconfHook } :
+
+assert (openssl != null) == (gnutls == null);
+
+let
+  version = "unstable-2018-10-08";
+  name = "openconnect_pa-${version}";
+in stdenv.mkDerivation {
```

```suggestion
in stdenv.mkDerivation rec {
```

`rec` allows you to self reference so you get rid of the `let in` and just put `version` and `name` in the set